packages/common/utils/src/clamp.spec.ts

```diff
@@ -1,17 +1,23 @@
 import { clamp } from './clamp';
 
-describe('clamp는', () => {
-  it('max 만 있을 때 잘 동작한다', () => {
+describe('clamp should', () => {
+  it('work well when only given max value', () => {
     expect(clamp(3, 5)).toBe(3);
     expect(clamp(10, 6)).toBe(6);
     expect(clamp(6, 10)).toBe(6);
   });
 
-  it('min, max 모두 있을 때 잘 동작한다', () => {
+  it('work well when both value was given', () => {
     expect(clamp(3, 5, 10)).toBe(5);
     expect(clamp(10, 6, 10)).toBe(10);
     expect(clamp(6, 10, 10)).toBe(10);
     expect(clamp(7, 5, 10)).toBe(7);
     expect(clamp(100, 5, 6)).toBe(6);
   });
+
+  it('throw error when bound1 is bigger than bound2', () => {
+    expect(() => clamp(3, 10, 5)).toThrowError();
+    expect(() => clamp(7, 12, 5)).toThrowError();
+    expect(() => clamp(100, 6, 5)).toThrowError();
```

Comment:
What about using [.toThrowErrorMatchingInlineSnapshot()](https://jestjs.io/docs/expect#tothrowerrormatchinginlinesnapshotinlinesnapshot) to check whether the thrown error is our desired error?

Changing `toThrowError()` to `toThrowErrorMatchingInlineSnapshot()` and running the test will generate the inline snapshot, making the test to fail when the function throws an error that is different from the desired one